dev-docs/show-outstream-video-ads.md

```diff
@@ -178,6 +183,48 @@ For more information, see the API documentation for:
 + [getAdserverTargetingForAdUnitCode]()
 + [renderAd]()
 
+## Option 3. With a Publisher-defined Renderer
+
+By default, an outstream bid response is accompanied by a video renderer in the form of a URL that points to a script containing the renderer software.  This software is used by Prebid.js to load and play the winning outstream bid.
+
+However, if you as a publisher would prefer to use a different renderer than the one provided in the bid response, you can specify that renderer on a per-outstream-ad-unit basis.  This renderer will be used to load and play the outstream ad, instead of the renderer that is supplied with the bid.  If both a publisher-supplied ad unit renderer and a bid renderer are present, the publisher-supplied renderer will be used.
+
+In order for Prebid.js to use the publisher-defined renderer, you need to set up an outstream ad unit with the `renderer` property.  It's an object with the following properties:
+
++ `url`: A URL pointing to the render script
++ `render`: A function that receives a bid parameter and invokes the renderer that tells Prebid.js how to use the script to render the bid
```

Comment:
Agreed!  I just rewrote it to use many fewer words, and a couple of inline comments in the example code snippet.  Let me know if it's an improvement.